Text Generation WebUI/src/server-config.ts

```diff
@@ -1,13 +1,33 @@
 import path from 'path';
-import { getEnvValeOrFail } from './utils';
-
-export const serverConfig = {
-  engineFolder: path.join(__dirname, '../text-generation-webui'),
-  privateKeyFilePath: `${__dirname}/private.pem`,
-  certificateFilePath: `${__dirname}/certificate.crt`,
-  port: getEnvValeOrFail('HTTPS_PORT') as number,
-  tlsKey: getEnvValeOrFail('TLS_KEY') as string,
-  tlsCert: getEnvValeOrFail('TLS_CERT') as string,
-  modelSizeThreshold: 100 * 1024 * 1024, // 100Mb,
-  configurationPath: process.env.CONFIGURATION_PATH || '/sp/configurations/configuration.json',
+import { config } from './config';
+import { getEnvValueOrFail } from './env-utils';
+
+export interface IServerConfig {
+  engineFolder: string;
+  privateKeyFilePath: string;
+  certificateFilePath: string;
+  port: number;
+  tlsKey: string;
+  tlsCert: string;
+  modelSizeThreshold: number;
+  configurationPath: string;
+}
+
+let serverConfig: IServerConfig;
+
+export const getServerConfig = (): IServerConfig => {
```

Comment:
why do we have to use wrapper fn?

Reply:
To avoid the error about missing HTTPS_PORT env var